www/include/configuration/configObject/service/formService.ihtml

```diff
@@ -621,22 +633,34 @@ jQuery(function() {
                             jQuery(elem).find("input[name^='macroValue']").val(tplValue);
                             jQuery(elem).find("input[name^='macroInput']").val(tplInput);
                             jQuery(elem).find('input[name^=macroFrom]').val("fromTpl");
-                            jQuery(elem).find("input[name^='macroInput']").css({'background-color' : '#ffcb6b',border : '1px solid #ffa600'});
-                            jQuery(elem).find("input[name^='macroValue']").css({'background-color' : '#ffcb6b',border : '1px solid #ffa600'});
+                            jQuery(elem).find("input[name^='macroInput']").css({'background-color' : '#ffcb6b',border : '1px solid #ffa600',color: 'var(--custom-macros-placeholder-font-color)'});
```

Comment:
It is possible to use a variable for the color #ffa600 ?

Reply:
@callapa it was like this before but yes it is possible to export it ! :)

Reply:
That was the purpose of the "dark mode" feature, right?